source/administration/changelog.md

```diff
@@ -2,6 +2,119 @@
 
 This changelog summarizes updates to [Mattermost Team Edition](http://www.mattermost.org/), an open source team messaging solution released monthly under an MIT license, and [Mattermost Enterprise Edition](https://about.mattermost.com/pricing/), a commercial upgrade offering enterprise messaging for large organizations.
 
+## Release v3.4.0
+
+Expected release date: 2016-08-16
+
+### Highlights
+
+Heroku Deployment
+
+Zapier App
+
+Improved Notifications and Status Indicators
+- Users can now control how often email notifications are sent
+- Users can now control whether push notifications are sent when they are online, offline or away
+- Users can now set how long desktop notifications stay onscreen for
+- Email notifications now include the channel name
+- (?) Push notifications are now cleared after the message is read somewhere else
+- /away, /online, /offline can now be used to manually set your status
+- Status indicators are now shown on the profile pictures in the centre channel and right hand side
+
+### Improvements
+
+#### Files and Images
+- PDFs now show a preview in the image previewer
```

Comment:
Maybe mention this works across mobile and desktop apps, just to highlight the value a bit, since PDF preview is built into web browsers these days...